packages/wallet-api-manifest-validator/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@ledgerhq/wallet-api-manifest-validator",
+  "version": "1.0.0",
+  "description": "This package checks if your manifest.json file meets the requirements for Ledger Live App manifest submission.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "license": "MIT",
+  "files": [
+    "/dist"
+  ],
+  "scripts": {
+    "build": "tsup src/ && tsup cli",
+    "build:src": "tsup src",
```

If you need to keep it I think it's better like that
```suggestion
    "build:src": "tsup src/index.ts --dts --format esm,cjs,iife",
```